practices/responsibility.md

```diff
@@ -0,0 +1,91 @@
+
+Responsibility is not for everyone. But it’s a powerful and deeply rewarding ship to navigate once embarked.
+
+![Responsibility](https://shiftyourfamilybusiness.com/wp-content/uploads/2018/05/ResponsibilityBlog-845x321.jpg)
+
+
+## Why You Might Want Responsibility
+
+Being responsible brings us many benefits. It can help us achieve our goals and objectives in any area of our life. Responsibility allows us to create principles, morals and helps us to lead your life. Being a responsible person helps us to:
+
+**Be more independent:** Assuming the consequences of our actions will help us make better decisions.
+**Be more reliable**: By being responsible, we gain other's trust and we will also gain confidence in ourselves
+**Be more honest:** When we tend to tell the truth and keep our promises, the people around us will believe us and see us as an honest person.
+
+
+
+## Problems Responsibility Solves
+
+- Establishes team accountability
+- Key to great team work
+- Quality of output is usually high as the tasks done by the team are mostly interdependent
+- An individual is committed to complete the tasks allocated to them
+- Problem solving is given importance rather than blaming others
+- Makes your team members feel important and appreciated
+- Helps an individual do more, faster
+
+
+## How to Implement Responsibility
+
+- **Start by Talking**: The first step is to talk to the individuals concerned
+- **Ensure Adequate Resources**: A sensible action is to ensure that your people have the resources they need to do their job . This might include providing training, equipment, access to mentors  and coaches , and suchlike.
+- **Communicate Roles, Responsibilities, and Objectives**: Your people also need to know clearly what their job roles and responsibilities are.
+- **Re-Engage People**: You take responsibility for your actions, simply because you have a deep sense of pride in what you're doing. The same will likely hold true for your people: by working on [re-engaging](https://www.mindtools.com/pages/article/newTMM_83.htm) them.
+
+
+**Tips for implementing Responsibility**
+1. Make sure that you have clear, accurate examples that you can cite when you provide feedback. 
+2. Take the [How Well Do You Develop Your People?](https://www.mindtools.com/pages/article/team-development.htm)  self-test to improve your team development skills.
+3. Sometimes, people don't take responsibility because they feel apathetic about their work. They can't see how their efforts tie into the "bigger picture." So, make sure that they understand how their work ties into the larger goals of the organization
```

Comment:
Please, rewrite this sentence since it is similar to a sentence from the Mindtools article: _Sometimes, people don't take responsibility because they feel apathetic about their work. They can't see how their efforts tie into the "bigger picture."_